doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -501,17 +501,17 @@ the merge message input will be automatically populated.
 Using .githubmap to Find a Reviewer's Email Address
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 If you cannot find the email address of the reviewer on his or her GitHub
-page, you can look it up in the **.githubmap** file, which can be found in
-the repository at **/ceph/.githubmap**.
+page, you can look it up in the ``.githubmap`` file, which can be found in
+the repository at ``/ceph/.githubmap``.
```

Comment:
I think you want to remove the first slash

Reply:
I have the opposite intuition, but I'm willing to be argued out of it. I thought that the initial slash here made it clear that the location begins at the root ``/`` of the repository's filesystem.